autogen/coding/local_commandline_code_executor.py

```diff
@@ -43,6 +44,8 @@ class LocalCommandlineCodeExecutor(BaseModel):
     the working directory, and a unique file is generated and saved in the
     working directory for each code block.
     The code blocks are executed in the order they are received.
+    Command line code is sanitized in order to prevent self destructive
```

Comment:
```suggestion
    Command line code is sanitized using regular expression match against a list of dangerous commands in order to prevent self destructive
```